### DISCUSSION TITLE
donation to magit is tax deductible?

### DISCUSSION THREAD

**Discussion by eychun** [magit/magit#4835] [Support] [answered]
Are donations to Magit considered tax deductible in the USA? Is it a 501c3? I'd like to file my donation with my company to get it matched, but I think it needs to be a donation to a non-profit charity. I'm not sure if Magit counts.

**Comment by tarsius** [comment, marked as answer]
I am going through old discussions and noticed that I never responded to this.

First, thanks for your support!

Magit isn't a 501c3. It's just me, Jonas. A human. :grinning:

(With some support from others of course. Nowadays mostly @kyleam.)
